### PR TITLE
Add point describing when to do releases

### PR DESCRIPTION
**Pull request by arj03**
This should take care of #8. Also with this in place I think the rule about no other branches needs to go. I'll do a separate PR for that.

### CODE REVIEW

**Review by christianbundy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,10 +34,11 @@ SSB-JS organization require discussion and consent.
 ### Repositories
 
 - There MUST be one Git repository for governance and one for each project.
-- The 'SSB-JS namespace' refers to the SSB-JS organizations on GitHub and Npm.
+- The 'SSB-JS namespace' refers to the SSB-JS organizations on GitHub and npm.
 - Repositories and build artifacts must be hosted in the SSB-JS namespace.
 - Repositories MUST have a 'main' branch.
 - Repositories MUST NOT have any other branches.
+- The latest commit on the 'main' branch SHOULD always be versioned and released on npm.
```
Comment (mixmix) [inline README.md:41]: and tags pushed to github

**Review by mixmix — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (christianbundy) [reply README.md:41]: :pray:

**Review by soapdog — APPROVED** [review]
LGTM

**Comment by christianbundy** [comment]
Thanks for taking the time to propose this @arj03, this is a great patch.

**Pull request merged**